notification/src/main/java/com/chronica/notification/data/constant/PriorityType.java

```diff
@@ -0,0 +1,15 @@
+package com.chronica.notification.data.constant;
+
+import lombok.Getter;
+
+@Getter
+public enum PriorityType {
```

Comment:
To powinno raczej być po prostu Priority. Ten enum to nie typ rpiorytetu tylko priorytet